serve.py

```diff
@@ -1,34 +1,110 @@
+    os.makedirs(logdir, exist_ok=True)
+LOG_SETTINGS = dict(
+    version=1,
+    disable_existing_loggers=False,
+    loggers={
+        "sanic.root": {"level": "INFO", "handlers": ["consolefile"]},
+        "sanic.error": {
+            "level": "INFO",
+            "handlers": ["error_consolefile"],
+            "propagate": True,
+            "qualname": "sanic.error",
+        },
+        "sanic.access": {
+            "level": "INFO",
+            "handlers": ["access_consolefile"],
+            "propagate": True,
+            "qualname": "sanic.access",
+        },
+    },
+    handlers={
+        "consolefile": {
+            "class": "logging.handlers.TimedRotatingFileHandler",
+            "when": 'D',
+            "interval": 7,
+            "backupCount": 10,
+            'filename': os.path.join(logdir, "console.log"),
+            "formatter": "generic",
+        },
+        "error_consolefile": {
+            "class": "logging.handlers.TimedRotatingFileHandler",
+            "when": 'D',
+            "interval": 7,
+            "backupCount": 10,
+            'filename': os.path.join(logdir, "error.log"),
+            "formatter": "generic",
+        },
+        "access_consolefile": {
+            "class": "logging.handlers.TimedRotatingFileHandler",
+            "when": 'D',
+            "interval": 7,
+            "backupCount": 10,
+            'filename': os.path.join(logdir, "access.log"),
+            "formatter": "access",
+        },
+    },
+    formatters={
+        "generic": {
+            "format": "%(asctime)s [%(process)d] [%(levelname)s] %(message)s",
+            "datefmt": "[%Y-%m-%d %H:%M:%S %z]",
+            "class": "logging.Formatter",
+        },
+        "access": {
+            "format": "%(asctime)s - (%(name)s)[%(levelname)s][%(host)s]: "
+                      + "%(request)s %(message)s %(status)d %(byte)d",
+            "datefmt": "[%Y-%m-%d %H:%M:%S %z]",
+            "class": "logging.Formatter",
+        },
+    },
+)
+
+if production:
+    app = Sanic("redirector", log_config=LOG_SETTINGS)
+else:
+    app = Sanic("redirector")
 CORS(app)
 
+
 @app.listener("before_server_start")
 async def init(app, loop):
-    app.sem = asyncio.Semaphore(100, loop=loop)
+    global sem
+    sem = asyncio.Semaphore(100)
 
 
 @app.route("/")
 async def main(request):
-    return response.redirect("https://www.dandiarchive.org/")
+    return response.redirect("https://gui.dandiarchive.org/")
+
 
 @app.route("/about")
 async def about(request):
     return response.redirect("https://www.dandiarchive.org")
 
 
-@app.route("/dandiset/drafts")
-async def get_draft_collection(request):
+@app.route("/dandiset")
+async def get_public_dashboard(request):
     """Redirect to gui draft collection
     """
-    return response.redirect("https://gui.dandiarchive.org/#/collection/5e59bb0af19e820ab6ea6c62")
+    return response.redirect("https://gui.dandiarchive.org/#/dandiset")
+
+
+@app.route("/portal")
```

Comment:
why to bother with this new additional route if `/` to point to the same?  if anything, would be nice to keep things consistent, e.g. either have it confusing `/gui` or RF to have `portal.dandiarchive.org` ... ;)
or here specifically -- underlying function is `about`.  Having some `/about` might make sense and should not "taxing" on API -- we always could have some `about`

Reply:
because `/` can change on a whim :)  but indeed portal is not necessary. i was just trying to minimize our own use of routes to rely on the redirector.

Reply:
is this route staying or going away?

Reply:
it was introduced in this PR. i don't have a strong feeling. @yarikoptic ?

Reply:
@yarikoptic - ping

Reply:
Well, I am in-line with my initial comment here -- I do not like adding "portal" since it has no other reflection of itself, and I do not see it needed ATM.